src/core/style/_variables.scss

```diff
@@ -3,6 +3,10 @@
 
 // Typography
 $md-body-font-size-base: rem(1.400) !default;
+$font-family: Roboto, 'Helvetica Neue', sans-serif !default;
+
+// Media queries
+$md-mobile-screen: 600px;
```

Comment:
I would go with using the main spectrum of xsmall to xlarge. While I was thinking about this, it occurred to me that we could also do something like this:

```scss
$md-xsmall: "max-width: 600px";

@media ($md-xsmall){
  card {
    color: red;
  }
}
```
What do you think?